application/templates/_shared/_breadcrumb.html

```diff
@@ -1,14 +1,23 @@
-{%  macro breadcrumb() %}
+{% macro render_breadcrumbs(breadcrumbs) %}
+  {% if breadcrumbs or breadcrumbs is not defined %}
```

Comment:
Just to check... the intention here is that `not defined` gets a default single breadcrumb to the homepage, but it's possible to explicitly set `breadcrumbs = none` to get rid of that if needs be?

If so, makes sense :+1:

Reply:
Yep, that's the intention 👍